File: c++/practica4.cpp
#include <iostream>
using namespace std;
#include <string>

int parentesis(string);

int main(){
  string a;
  cout<<"Ingresa parentesis (): ";
  cin >> a;
  int n = parentesis(a);
  cout<<"El numero de () que hay es: " << n<<endl;
  return 0;
}

int parentesis(string a){
  int j = 0; 
  int n = sizeof(a);
  for(int i = 0 ; i < n; i ++ ){
    if((a[i] == '(') && (a[i+1] == ')')){ 
      j ++; 
      i ++; 
    }
  }
  return j;
}

File: python/practica4.py
def parentesis(a):
    j = 0
    n = len(a)
    for i in range(n-1):
        if(a[i] == '(') and (a[i+1] == ')'):
            j = j + 1
            i = 1 + 1
    return j

a = input('Ingresa parentesis (): ')
n = parentesis(a)
print('El numero de parentesis () son: ', n)

